CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # CHANGES
 
+## [TBD] (TBD)
```

Comment:
Actually next release should be `0.7.10` due to various reasons we didn't published it despite stating concrete date thre.

Could you please move it to 0.7.10, sorry about it

Reply:
NP. Done.